### ISSUE TITLE
Set margin between sidebar and body

### ISSUE THREAD

**Issue by plammens** [spagnuolocarmine/TwentySecondsCurriculumVitae-LaTex#10]
Would be a nice feature to have.

**Comment by saifer92** [comment]
Agree. Also, don't know if is wanted, but the propic isn't centered in the sidebar.

**Comment by KasparJohannesSchneider** [comment]
I personally don't like how small the margin is between the text of "other information" and the grey sidebar.

![image](https://user-images.githubusercontent.com/16350820/109084242-b4feb580-7707-11eb-8730-46df1b50f108.png)
